examples/gojs/stars/main.go

```diff
@@ -0,0 +1,36 @@
+package main
```

Comment:
@mathetake fyi this might be a decent one to use for some perf curiosity tests as it makes 7.7MB of wasm that doesn't execute very fast due to so many host functions (and allocations within them)